src/logfile_values.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+#
+# logfile_values.py
+#
+# Copyright (c) 2017, InnoGames GmbH
+#
+"""
+logfile_values.py -- a python script to find metrics values in log file
+
+This script is using last line of log file to get metric value by column number
+
+python log_values.py --name=n` --column=1 --name=n2 --column=2
+"""
+
+from argparse import ArgumentParser
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='logfile_values')
+    parser.add_argument('--file', default='1.log')
+    parser.add_argument(
+        '-n',
+        '--name',
+        help='Graphite metric name',
+        default=None,
+        action='append',
+        dest='names', )
+    parser.add_argument(
+        '-c',
+        '--column',
+        help='Number of the column with metric value',
+        default=None,
+        action='append',
+        dest='columns', )
+    args = parser.parse_args()
+    if len([x for x in (args.names) if x is not None]) != len(
+        [x for x in (args.columns) if x is not None]):
```

Comment:
This would be `len(args.name) == len(args.columns)` without the `default`s.

Reply:
Generally `[x for x in (args.names) if x is not None]` can be replaced with:
`filter(None, args.names)`